src/sentry/utils/concurrent.py

```diff
@@ -12,6 +12,27 @@
 logger = logging.getLogger(__name__)
 
 
+def execute(function, daemon=True):
```

Comment:
I like this a lot

Reply:
daemon=True might not be the right call here though

Reply:
I think daemon is safe for the one current use, but I wonder if it should default to `False` and we should opt in (to defend against future expectations or something)? Really minor though.

Reply:
I think I agree, I'll update this to a more generally useful default.